src/olympia/lib/settings_base.py

```diff
@@ -1606,3 +1606,7 @@ def read_only_mode(env):
 SOCKET_LABS_HOST = env('SOCKET_LABS_HOST', default='https://api.socketlabs.com/v2/')
 SOCKET_LABS_TOKEN = env('SOCKET_LABS_TOKEN', default=None)
 SOCKET_LABS_SERVER_ID = env('SOCKET_LABS_SERVER_ID', default=None)
+
+# Set to True in settings_test.py
+# This controls the behavior of migrations
+TESTING_ENV = False
```

Comment:
Do we still need https://github.com/mozilla/addons-server/blob/f6569245aa5d6e4b89ead29d01fc9c74bb1635a7/settings.py#L198-L199 if it's defined there?

Reply:
good catch.